service/debugger/debugger.go

```diff
@@ -176,8 +191,50 @@ func (d *Debugger) checkGoVersion() error {
 	return goversion.Compatible(producer)
 }
 
+func verifyBinaryFormat(exePath string) error {
+	f, err := os.Open(exePath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	fi, err := f.Stat()
+	if err != nil {
+		return err
+	}
+	// check that the argument is an executable file
+	if (fi.Mode()&0111) == 0 {
```

Comment:
windows executables don't have the executable bit set, this is what's causing the test to fail.

Reply:
Fixed, I was looking at the sources of exec.LookPath which checked the unix executable bit on godoc but it went straight to the _unix.go file and I wasn't paying attention. I was planning to keep working on this last night so it wasn't still failing in the morning, but it was getting late.